build.assets/Dockerfile

```diff
@@ -36,14 +39,18 @@ RUN apt-get update -y --fix-missing && \
         make \
         mingw-w64 \
         net-tools \
+        openssh-client \
```

Comment:
why do we need `openssh-client`?

Reply:
I added `--no-install-recommends` to the `apt-get` commands in an effort to make the layer size a bit smaller; this had the side effect of no longer installing `openssh-client`, which was seemingly a recommendation included when installing another package. It was causing unit tests to fail as they need the `scp` binary.